### PR TITLE
Fail if ovirt_repositories_use_subscription_manager is set and OS is not RHEL

### PR DESCRIPTION
**Pull request by KKoukiou**
Fixes https://github.com/oVirt/ovirt-ansible-repositories/issues/4

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 ---
+- name: Perform parameter compatibility check
+  fail:
+    msg: "Subcription manager should run only when ansible_distribution == 'Red Hat Enterprise Linux'"
```
Comment (mwperina) [inline tasks/main.yml]: I'd rephrase, it seems to me too "developer specific" :-)
```
msg: "Subcription manager could be used only on Red Hat Enterprise Linux"
```

**Review by mwperina — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (KKoukiou) [reply tasks/main.yml]: Done

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks!

**Review by machacekondra — APPROVED** [review]
